Documentation/MAX32672.html

```diff
@@ -55,20 +55,27 @@ <h2>Example Applications</h2>
 		<ul>
 			<li><a href='../Examples/MAX32672/ADC/README.md'>ADC</a> - Analog-to-Digital Converter</li>
 			<li><a href='../Examples/MAX32672/AES/README.md'>AES</a> - Advanced Encryption Standard</li>
+			<li><a href='../Examples/MAX32672/ARM-DSP/README.md'>DSP</a> - Digital Signal Processing</li>
 			<li><a href='../Examples/MAX32672/Comparator/README.md'>Comparator</a></li>
 			<li><a href='../Examples/MAX32672/CRC/README.md'>CRC</a> - Cyclic Redundancy Check</li>
 			<li><a href='../Examples/MAX32672/CTB_AES/README.md'>CTB_AES</a> - Cryptographic Toolbox Advanced Encryption Standard</li>
 			<li><a href='../Examples/MAX32672/DMA/README.md'>DMA</a> - Direct Memory Access</li>
```

Comment:
Missing Display example.

Reply:
Correct.  It needs a README.md file to link to.  I have another task on the TODO list to clean up the few examples that are missing the readme.